lib/payload.json

```
@@ -137,6 +141,28 @@
             }
           },
           "additionalProperties": false
+        },
+        "water": {
+          "type": "object",
+          "properties": {
+            "depth": {
+              "description": "Depth of water measurement (cm)",
+              "$ref": "#/definitions/depth"
+            },
+            "temperature": {
+              "description": "Water temperature (°C)",
+              "$ref": "#/definitions/temperature"
+            },
+            "pH": {
+              "description": "Water pH level",
+              "$ref": "#/definitions/pH"
+            },
+            "salinity": {
+              "description": "Salt (mg) per litre (ppm)",
```

We're following en-US
```suggestion
              "description": "Salt (mg) per liter (ppm)",
```